crates/biome_text_size/src/range.rs

```diff
@@ -83,8 +83,13 @@ impl TextRange {
     /// assert_eq!(&text[range], "23456")
     /// ```
     #[inline]
-    pub fn at(offset: TextSize, len: TextSize) -> TextRange {
-        TextRange::new(offset, offset + len)
+    pub const fn at(offset: TextSize, len: TextSize) -> TextRange {
+        TextRange {
+            start: offset,
+            end: TextSize {
+                raw: offset.raw + len.raw,
+            },
+        }
     }
```

Comment:
This is not required anymore. I kept it because this make all constructors `const fn`. This seems more consistent with the API of `Textrange` because there are already many methods that are `const fn`.